packages/default-theme/style/tabbar.css

```diff
@@ -93,5 +93,4 @@
   min-width: 125px;
   border: none;
   box-shadow: 1px 1px 2px rgba(0, 0, 0, 0.3);
-  transform: translateX(-40%) translateY(-58%);
```

Comment:
I removed this because it's always overridden by

https://github.com/jupyterlab/lumino/blob/3f0be5cd32c1ff892ed887e60409c1a3b37a9238/packages/dragdrop/src/index.ts#L229-L234